src/ui/atoms/thumbnail/Thumbnail.tsx

```diff
@@ -5,7 +5,7 @@ import "./Thumbnail.css";
 interface ThumbnailProps {
   className?: string;
   bgColor?: string;
-  size?: "md" | "sm";
+  size?: "md" | "md-3x5" | "sm" | "sm-3x5";
```

Comment:
Hmm. I wonder if we can introduce this aspect ratio in some other way? Should we just make the thumbnail fill the container and set the container width/height based on flexbox or css grid?

Reply:
I revert these changes as these are not needed anywhere. Maybe we can figure about this when we actually need them.